compiler_opt/rl/corpus.py

```diff
@@ -34,77 +35,93 @@ def build_modulespecs_from_datapath(
     additional_flags: Tuple[str, ...] = (),
     delete_flags: Tuple[str, ...] = ()
 ) -> List[ModuleSpec]:
-  module_paths: List[str] = _load_module_paths(data_path)
-
-  has_thinlto: bool = _has_thinlto_index(module_paths)
+  # TODO: (b/233935329) Per-corpus *fdo profile paths can be read into
+  # {additional|delete}_flags here
+  metadata: Dict[str, any] = _load_metadata(
+      os.path.join(data_path, 'metadata.json'))
+  module_paths = metadata['modules']
+  if len(module_paths) == 0:
+    raise ValueError(f'{data_path}\'s metadata contains no modules.')
+
+  has_thinlto: bool = metadata['has_thinlto']
+
+  # Future: cmd_override could have per-module overrides if needed
+  if 'global_command_override' in metadata:
+    if metadata['global_command_override'] == ['Please', 'fill', 'options']:
```

Comment:
nit: instead of "please fill options", maybe you can have a token like "UNSPECIFIED" or something like that (more "formal")

Reply:
or '<UNSPECFIED>'

Reply:
were you going to suggest something else?

Reply:
yes - sorry. <UNSPECIFIED> (not sure what happened)

Reply:
got it, thanks for the review!